src/syntax.ml

```diff
@@ -38,8 +38,8 @@ and p_term_aux =
   | P_Wild
   (** Wildcard (place-holder for terms). *)
   | P_Meta of strloc * p_term array
-  (** Meta-variable with the given environment. *)
-  | P_Patt of strloc * p_term array
+  (** Named or unnamed Meta-variable with the given environment. *)
```

Comment:
Meta-variable -> Pattern-variable

Reply:
Oups, I modified the wrong comment. Fixed.